interface_config.js

```diff
@@ -189,6 +189,12 @@ var interfaceConfig = {
      */
     AUTO_PIN_LATEST_SCREEN_SHARE: 'remote-only'
 
+    /**
+     * The link to the user documentation.
+     */
+    // HELP_LINK: 'https://docs.example.com/video-meetings.html',
```

Comment:
Why don't we have a static HTML file or something on jitsi.org by default?

Reply:
I don't think we currently have a user documentation that we can use.